handlers.go

```
@@ -45,10 +45,84 @@
+			log.Info().Msgf("failed to write response: %v\n", err)
+			requestError(w, http.StatusInternalServerError)
+			return
+		}
+	})
+}
+
+// Similar handler, but adapted specifically to accommodate org profiles.
+// This is partially done due to the fact that the returned props format is
+// already in use by the plugin, and it would be good to continue supporting
+// it during the transition to allow for the use of profiles.
+func handlePostOrgGitCredentials(tokenVendor vendor.ProfileTokenVendor) http.Handler {
```

golangci-lint (1.64.8)

func `handlePostOrgGitCredentials` is unused

(unused)